helix-view/src/editor.rs

```diff
@@ -243,11 +243,28 @@ where
     Ok(chars)
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case", default, deny_unknown_fields)]
+pub struct ScrolloffConfig {
+    pub horizontal: usize,
+    pub vertical: usize,
+}
+
+impl Default for ScrolloffConfig {
+    fn default() -> Self {
+        Self {
+            horizontal: 5,
+            vertical: 5,
+        }
+    }
+}
+
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(rename_all = "kebab-case", default, deny_unknown_fields)]
 pub struct Config {
     /// Padding to keep between the edge of the screen and the cursor when scrolling. Defaults to 5.
-    pub scrolloff: usize,
+    // pub scrolloff_horizontal: usize,
```

Comment:
Hm... did you forgot to remove the old entry here?

![peepoThonk](https://github.com/user-attachments/assets/dff203d6-dd26-4797-8d1f-9d3dc0054539)